File: Front/CampEventos-App/src/app/models/Apresentador.ts
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";

export interface Apresentador {

        id : number;
        nome  : string;
        miniCurriculo  : string;
        imagemURL : string;
        telefone  : number;
        email : string;
        redesSociais : RedeSocial[];
        apresentadoresEventos : Evento[];
}

File: Back/src/CampEventos.Domain/Apresentador.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using CampEventos.Domain.Identity;

namespace CampEventos.Domain
{
    public class Apresentador
    {
        public int Id { get; set; }
        public string MiniCurriculo  { get; set; }
        public int UserId  { get; set; }
        public User User { get; set; }
        public IEnumerable<RedeSocial> RedesSociais  { get; set; }
        public IEnumerable<ApresentadorEvento> ApresentadoresEventos { get; set; }
    }
}